source/write-operations/insert-documents.txt

```diff
@@ -0,0 +1,208 @@
+     - | If set to ``True``, allows the write to opt out of
+         :manual:`document-level validation </core/schema-validation>`. 
+       | Defaults to ``False``.
+
+   * - ``toString``
+     - | If used, returns a string representation of the object.
+
+   * - ``comment``
+     - | A comment to attach to the operation. For more information, see the :manual:`insert command
+         fields </reference/command/insert/#command-fields>` guide in the
+         MongoDB Server manual for more information.
+
+The ``InsertManyOptions()`` method accepts the preceding optional parameters,
+as well as the optional ``ordered`` property:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Property
+     - Description
+
+   * - ``ordered``
+     - | If set to ``True``, the driver sends documents to the
+         server in the order provided. If an error occurs, the driver
+         and server cancel all remaining insert operations.
+       | Defaults to ``True``. 
+
+Example
+~~~~~~~
+
+The following code uses the ``insertMany()`` method as well as the
+``InsertManyOptions()`` method to insert three new
+documents into a collection. Because the argument is
+``bypassDocumentValidation(true)``, this insert operation bypasses 
+document-level validation.
+
+.. code-block:: java
+
+   Document doc1 = new Document("name", "Mongo's Burgers");
+   Document doc2 = new Document("name", "Mongo's Pizza");
+   Document doc3 = new Document("name", "Mongo's Coffee");
+
+   List<Document> documents = new ArrayList<Document>();
+
+   documents.add(doc1);
+   documents.add(doc2);
+   documents.add(doc3);
+
+   ObservableSubscriber<InsertManyResult> insertManySubscriber = new SubscriberHelpers.OperationSubscriber<>();
+   restaurants.insertMany(documents, new InsertManyOptions().bypassDocumentValidation(true)).subscribe(insertManySubscriber);
+   InsertManyOptions.await();
+
+.. Additional Information
+----------------------
+
+.. TODO:
+.. For runnable code examples of inserting documents with {+driver-short+}, see
+.. :ref: ADD HERE
+
+API Documentation
+~~~~~~~~~~~~~~~~~
+
+To learn more about any of the methods or types discussed in this
+guide, see the following API Documentation:
+
+- `insertOne() <https://mongodb.github.io/mongo-java-driver/5.1/apidocs/mongodb-driver-reactivestreams/com/mongodb/reactivestreams/client/MongoCollection.html#insertOne(com.mongodb.reactivestreams.client.ClientSession,TDocument)>`__
+- `insertMany()
+  <https://mongodb.github.io/mongo-java-driver/5.1/apidocs/mongodb-driver-reactivestreams/com/mongodb/reactivestreams/client/MongoCollection.html#insertMany(com.mongodb.reactivestreams.client.ClientSession,java.util.List)>`__
+- `InsertOneOptions <https://mongodb.github.io/mongo-java-driver/5.1/apidocs/mongodb-driver-core/com/mongodb/client/model/InsertOneOptions.html>`__
+- `InsertManyOptions <https://mongodb.github.io/mongo-java-driver/5.1/apidocs/mongodb-driver-core/com/mongodb/client/model/InsertManyOptions.html>`__
+
```

Comment:
S: I would add `BsonValue` here, since it's used as a parameter type in one of the methods